### DISCUSSION TITLE
Which film won the Oscar for Best Picture in 2020?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#337] [Galaxy Trivia] [answered]
**Hint:** 🦠

**Comment by GameExplorer** [comment]
Parasite

**Comment by thuongtruong1009** [comment, marked as answer]
The 92nd Academy Awards ceremony was held in 2020, and the movie that won the Oscar for Best Picture was "Parasite", directed by Bong Joon-ho. "Parasite" is a South Korean dark comedy thriller that explores issues of social inequality and class conflict. The film made history as the first non-English language movie to win the Best Picture award in the 92-year history of the Academy Awards.

**Comment by peterbaccaro** [comment]
The Oscar for Best Picture in 2020 was awarded to the movie "Parasite." Directed by Bong Joon-ho, the South Korean film made history as the first non-English language film to win the award.
